birdhouse/config/proxy/default.env

```diff
@@ -0,0 +1,39 @@
+# All env in this default.env can be overridden by env.local.
+
+# Folder inside "proxy" container to drop extra monitoring config
+export CANARIE_MONITORING_EXTRA_CONF_DIR="/conf.d"
+
+# Timeout for reading a response from the proxied server.
+# Any WPS processes taking longer than this should use async mode.
+export PROXY_READ_TIMEOUT_VALUE="240s"
+
+# Content of "location /" in file config/proxy/conf.d/all-services.include.template
+# Useful to have a custom homepage.
+export PROXY_ROOT_LOCATION="return 302 https://\$host/jupyter/hub/login;"
+
+export INCLUDE_FOR_PORT_80='$([ x"$ALLOW_UNSECURE_HTTP" = x"True" ] && echo "include /etc/nginx/conf.d/all-services.include;" || echo "include /etc/nginx/conf.d/redirect-to-https.include;")'
```

Comment:
So you like this delayed eval feature? :D

Had delayed eval not exist, probably `pre-docker-compose-up` might be able to do this.